ext/vulnsrc/oracle/oracle.go

```diff
@@ -115,7 +150,7 @@ func (u *updater) Update(datastore database.Datastore) (resp vulnsrc.UpdateRespo
 		r := elsaRegexp.FindStringSubmatch(line)
 		if len(r) == 2 {
 			elsaNo, _ := strconv.Atoi(r[1])
-			if elsaNo > firstELSA {
+			if compareELSA(elsaNo, firstELSA) == 1 {
```

Comment:
Usually you'd test for `> 0` because this kind of comparison functions typically only offers the sign and 0 guarantees (C / Java) and return a distance/difference between the two (at least for the first part that is different).

i.e. ```return len(lstr) - len(rstr)```